lode-ui/src/App.js

```diff
@@ -1,16 +1,23 @@
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import "./App.css";
+import { BrowserRouter as Router } from "react-router-dom";
+import { ChakraProvider, Grid, GridItem } from "@chakra-ui/react";
 import ModelChooser from "./ModelChooser";
 import ModelViewer from "./ModelViewer";
+import "./App.css";
```

Comment:
I'd suggest we use emotion consistently and remove the last remaining `.css` styling? 😄